tests/dependency/grpc/test_sanity.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import os
+import time
+from pinecone import ServerlessSpec
+from pinecone.grpc import PineconeGRPC
+
+@pytest.fixture
+def index_name():
+    return 'test-sanity-rest'
+
+@pytest.fixture
+def client():
+    return PineconeGRPC(api_key=os.environ.get('PINECONE_API_KEY'))
+
+@pytest.fixture(autouse=True)
+def cleanup(index_name, client):
+    yield
+    if index_name in client.list_indexes().names():
+        client.delete_index(name=index_name)
+
+class TestSanityRest:
+    def test_sanity(self, index_name, client):
+        if index_name not in client.list_indexes().names():
+            client.create_index(
+                name=index_name, 
+                dimension=2, 
+                spec=ServerlessSpec(cloud="aws", region="us-west-2")
+            )
+        else:
+            pytest.fail('Index ' + index_name + ' already exists')
+        
+        idx = client.Index(index_name)
+        idx.upsert(vectors=[
+            ('1', [1.0, 2.0]), 
+            ('2', [3.0, 4.0]),
+            ('3', [5.0, 6.0])
+        ])
+
+        # Wait for index freshness
+        time.sleep(30)
```

Comment:
Is grpc freshness a bit quicker than rest? We're using 30 seconds here and 60 for rest.

Reply:
I don't think it's quicker. Just these aren't enabled in CI yet.